src/scopes.ts

```diff
@@ -617,6 +618,13 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     type: ScopeType.Modify,
     products: [TranscendProduct.Assessments, TranscendProduct.DataMapping],
   },
+  [ScopeName.ViewPreferences]: {
+    title: 'View Preferences',
+    dependencies: [],
+    description: 'View preferences for users or organizations',
+    type: ScopeType.View,
+    products: [TranscendProduct.Admin],
```

Comment:
I wasn't really sure what this should be if it's a scope that spans the AD. Should I have listed them all out?